src/WebApp/Api/Webfinger/WebfingerEndpoints.cs

```diff
@@ -1,6 +1,6 @@
 using Microsoft.AspNetCore.Mvc;
 
-namespace Smilodon.WebApp.Api.Admin;
+namespace Smilodon.WebApp.Api.Webfinger;
```

Comment:
Should we try to make everything `WebFinger` for consistency?

Reply:
Yes, you're right. That's how the spec has it. Want to send that as a separate PR?

Reply:
Sure, can do. I also wanted to make all the property names consistent with the Microsoft Docs, which specify PascalCase for record properties, I can do a naming PR for that stuff.